docs/draw.io/data-model.drawio.xml

```diff
@@ -1 +1 @@
\ No newline at end of file
+<mxfile host="app.diagrams.net" modified="2022-06-08T04:35:59.930Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/101.0.0.0 Safari/537.36" etag="AA23yNE43AE_8xSmd7Wx" version="19.0.1" type="device"><diagram id="dun09WMIp-Qr-0l5YfAu" name="Page-1">7Vxrc5s4FP01ntndmWR4+vExTuK0m/SxcXe33W8yKJgGIyrk2O6vXwkkXhKOM7HBqUlmEusihHTP4dwrCdwzLxfrGwyi+QfkwqBnaO66Z171DGMwNOhfZtikBmtgpQYP+25q0nPD1P8JuVHj1qXvwrhUkSAUED8qGx0UhtAhJRvAGK3K1R5QUL5qBDwoGaYOCGTrv75L5ql1aGu5/R30vbm4sq7xIwsgKnNDPAcuWhVM5nXPvMQIkfTTYn0JA+Y74Zf0vEnN0axjGIZklxO+j76aK22yGhL77tbVoy833/47M9NWnkCw5APmnSUb4QEPo2UkX4xf/wliAtcqKMBMtJCPlrIEogUkeEPr8bMs7p9NGfhV7m1D+HRe9jRHmSPsZS3nTqAfuB9e4BP9eZ/MyYI2eaX3zDFzgE/5cgdmMPiMYp/4KKTHZogQtKAVQOB7zOBQ10FMDQGrOQbOI/Ns6F6iAOGkWfMh+Sk0esHPJSii1phg9JjR0MgshRY0bahNNHZEsI0VXBDPocsL9EjEhrFYe+yOPfdRPDj3KWjxuROgJa02ltHeSp4qBWSo2wPTUIDZDwjzQwTCEqr9H0t2L45nGTRnTurZC1oPe7PfDNvuGbQfWvHD74nDqLKE5CxO9ItVN4bROjkgWqWfPPb/z/f3F6ILdERpL9JDEs2oW5m9wLYy4CEKac3xgx8EFZNEuiqjFr7rssuMV3OfwGkEHHbNFWUEtSWDzwjDRsaF2Rjycolyg8lkcgjWiBhSFghbFoi+glLmoRhltS+Z2YDrNdM0VbfZwZxid5q5G/utGg7UY63J4OqjUZMa2t+7hmoFGdUKSmpo9WIqKemNT+bL2blDGXPKevpyRlXUw9Ylgg0aFVRVwpUCylyTyKPwTs4B6if2UzQpeFghkm6oiKQX+JNe8aj5U5IpLwBxfCzEGlbDkqWQLplZxqGYNTiCUG0/G6obVvNhF6p34/+ghgNbpzcNgzk6jtDMjwV+CM/EwNlR7XxUF7dFP2c4l9s+WDAihLOY/ZtCTH2vrHe60X53Uu48e7IVjD1YsDctCSfoenDKiwiTOfJQCILr3FrxaF7nDjHhSKD9DgnZcD+DJUFl4HcFAYbuBVs0zKlALROfDfFKU4LEo0KMltiBW8atc5UlAHtwG8C8QeaUrfBiGADiP5WXKfcPl90GXNS7ePOVnX9ui+I33lxSuFqXSpsX3mtHALPRPyqcRb8LgeR9GBMQsIvTgF8lQbzyFwFIXOfM/cC9Axu0ZOOl5ziPojSeI+z/pCCAXIEBFhCZWqnGlJ3JAcgZVLl7dUuUeV80ZeKAIYtQnwWAWma6AzERXalNIST5r+NRMfnePYeslerqqk7flnNF1VKXvi2B5le7Zzlr6FGf1V9ODg27zANBQJ0UAgLHDLRYomI2zlewU96moPXZLWTqzPdO4FOv38JNwUbhxdCdQgdDUjB/9zF4hygJqowWoAfwgbwgVaAZiOOH3l1y1pVVYatRm+wmdOSTiJTxBJBCmRJJELXCPv2C/e6eMWS39vM83JQRfo53o4Mt2ssrTtPlLHawH3Vq1JIambuy4pdXI0NeViipkZfMb4rh871bUaBLhWDlylRpKZ0HXUQRbcacnI5wZSrwhoRLXly5hxGabkKHCgqBnXK1oFy2agflNJVLXi4qKVdcCLJMs05GaYY7U+xYlMY0JXhOYh1FOH5/82Z+6mfkJ3tNHH6zeldXtSGd4fOz9n+rinEWbtWbcjbQxZImYom0p6VQd121grr1WYtfIZiY8rZtKZj4dQlwOlsvmSLsP9HOlhPiFZzNEXqUpvBLHJxOaMp04FWhSUlQ7WCxSbXv2m3os2s2taH/Yp3TzVFZeHR5795QCM/B9u5N1T51R6K3RSJRbo1E9fvjHYneCImyDKwtEllyKt4sif7oSPRaEpmavJfWLIlUb9p0JDpqEpnVV5NETtLW84yW6t2DLpy9LRIpEutmSdSX0Gp25TCZV0seTyj6/Mph/mhQr/hgUP6cUM2jQbVIPrviKLb7io/03F//+IBuifdxeDv7MNv8NX335YN4senQK5PVJ/osEdoaWpm0utn9mxOhTE1EOtR6Tt3N7t8ciaqRLNvcai2SjdqIZHt4OPVVEXC/kWzrfuiRPJxqtz0B78Ti1RNwS/HiQaMRR7zl2E3A3zCJBs2t4ihT/FZeXqkLONmxmoCzl0CybabzbBwRW8TNxxF4T35urMvNxxAZf0c3w0//WLOW0Fv75Gvhc2G6Sks5dKzwMuQqXj0slHZDwG3rZEG6P91LWOI5WsyWcY24FRDZRypsjs7tyoxKliZLIU3WHqRJSW7jVyH361Z4XnpTbH3fZEeB2/sSj61XZlrmwZZ4lFxqe6/iNPNtxZcNSOTdeYVH/UUDh0mVlByq/9KYLt0+Vg71qxwyD8UhWsy/+TCVrfzrI83r/wE=</diagram></mxfile>
```

Comment:
If you uncheck this, it will make xml un-compressed, which should be better for git version control.

<img width="400" alt="image" src="https://user-images.githubusercontent.com/105693507/172536368-361ca386-0c7c-4e90-a810-7d28e36fd630.png">

Reply:
it is in the preference settings of the the drawio app

Reply:
Can't comments in the jpg image, but just curious is the "1__*" relation between "Installation" and "GithubServerApp" a concret FK constraint or just a virtual relation?

Reply:
Yep, good suggestion.  We should probably add a foreign key to the GithubServerApp table to the Installation.  We need to clean up our DB schema frankly, including removing duplicate installations and a better foreign key between installations and subscriptions.